### PR TITLE
Feature/AUT-1778/introduce allow skiping attribute on items

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-1778

How to test:

- set in test Authoring Allow Skipping checkbox for items
- save
- go to Item Preview
- skip items with Allow Skipping without responding to them

### CODE REVIEW

**Review by bartlomiejmarszal — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: models/test/mapper/TestPreviewMapper.php [hunk 1]
```diff
@@ -102,6 +102,7 @@ public function map(AssessmentTest $test, Route $route, TestPreviewConfig $confi
                     'flagged' => false,
                     'viewed' => false,
                     'categories' => $itemRef->getCategories()->getArrayCopy(),
+                    'allowSkipping' => $itemRef->getItemSessionControl()->doesAllowSkipping()
```
Comment (bartlomiejmarszal) [inline models/test/mapper/TestPreviewMapper.php:105]: This variable should be added to `\oat\taoQtiTestPreviewer\test\unit\models\test\factory\TestPreviewMapperTest`

**Review by bartlomiejmarszal — CHANGES_REQUESTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by Silvia-loza — COMMENTED** [review]
LGTM and works fine in local with terre test preview, however throws error when changing config to solar preview. 
image
image

**Review by Silvia-loza — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not` master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Review by bartlomiejmarszal — COMMENTED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
LGTM

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**